docs/references/system_requirements/global.md

```diff
@@ -3,40 +3,34 @@ sidebar_label: "Global System Requirements"
 title: ""
 ---
 
-## Kubernetes Cluster Requirements
+## System Requirements
 
-For the Epinio server, and related deployments we recommend to consider the following resources:
+Please consider the following system requirements to be minimal, additional resources may be required for additional workload (apps).
 
-- x86_64, ARM32, or ARM64 architecture
-- Supported operating systems
-  - Linux
-    - [K3s](../../howtos/install_epinio_on_k3s.md)
-    - [K3d](../../howtos/install_epinio_on_k3d.md)
-    - [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)
-    - [RKE2](../../howtos/install_epinio_on_rke.md)
-  - Windows
-    - [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)
-  - MacOS
-    - [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)
-- Kubernetes versions 1.20 .. 1.23
-- 2-4 VCPUs
-- 8GB RAM (system memory + 4GB)
-- 10GB Disk space (system disk + 5GB)
+| Component | Description |
+| --- | --- |
+| OS / Kubernetes *) | Linux: [RKE2](../../howtos/install_epinio_on_rke.md), [K3s](../../howtos/install_epinio_on_k3s.md),   [K3d](../../howtos/install_epinio_on_k3d.md), [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)<br/>Windows, MacOS: [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md) |
+| CPU | 2-4 vCPUs |
+| Memory | 8GB RAM (system memory + 4GB) |
+| Storage | 10GB Disk space (system disk + 5GB) |
 
-In addition, extensive requirements for your workload (apps) would add to that.
+*) Linux: x86_64, arm64 and s390x; MacOS: x86_64, arm64 (epinio CLI only); Windows: x86_64
```

Comment:
<sup>1</sup> Linux: x86_64, arm64 and s390x; MacOS: x86_64, arm64 (epinio CLI only); Windows: x86_64

```suggestion
<sup>1</sup> Linux: x86_64, arm64 and s390x; MacOS: x86_64, arm64 (epinio CLI only); Windows: x86_64
```